projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-full-access-view/case-full-access-view.component.ts

```diff
@@ -275,11 +285,44 @@ export class CaseFullAccessViewComponent implements OnInit, OnDestroy, OnChanges
       }
       matTab = this.tabGroup._tabs.find((x) =>
         x.textLabel.replace(CaseFullAccessViewComponent.EMPTY_SPACE, '').toLowerCase() ===
-                                hashValue.replace(CaseFullAccessViewComponent.EMPTY_SPACE, '').toLowerCase());
+        hashValue.replace(CaseFullAccessViewComponent.EMPTY_SPACE, '').toLowerCase());
       if (matTab && matTab.position) {
         this.tabGroup.selectedIndex = matTab.position;
       }
     }
+
+  }
+
+  public getAccessType(): string {
+    let accessType: string;
+    const access_granted = this.caseDetails.metadataFields.find(metadataField =>
+      metadataField.id === CaseViewerComponent.METADATA_FIELD_ACCESS_GRANTED_ID);
+    if (access_granted.value.includes('SPECIFIC')) {
+      accessType = 'SPECIFIC'
+    } else if (access_granted.value.includes('CHALLENGED')) {
+      accessType = 'CHALLENGED'
```

Comment:
Very minor comment - Can we add semi-colons to the two accessType settings above please?